app/src/main/res/values/strings.xml

```diff
@@ -29,6 +29,12 @@
     <string name="mad_website_desc_string">www.mad.wi.zut.edu.pl</string>
     <string name="mad_facebook_string">Facebooku:</string>
     <string name="mad_facebook_website_string">https://www.facebook.com/MADWIZUT</string>
+    <string name="authors">Autorzy:</string>
+    <string name="authors_desc">Sara Anku \nMichał Bednarski \nDawid Heyman \nBartosz Kozajda \nMarek Macko \nDamian Malarczyk \nWaldemar Sambor</string>
+    <string name="ssa_patron">Opiekun koła naukowego:</string>
+    <string name="ssa_patron_desc">dr inż. Radosław Maciaszczyk</string>
+    <string name="legal_notices_title">\nLegal notices</string>
```

Comment:
- Use `android:layout_marginTop` instead of `\n`at start
- Maybe merge `legal_notices_title` with `legal_notices`, you can use `<b>Legal notices</b>\n\n`
- Put newline in xml after `\n` in multiline strings (`authors_desc`, `legal_notices`), putting `<string>` tags in separate lines as text.
- Also, I think it'd be better if legal notices used same margins as rest of texts